controllers/front/OrderOpcController.php

```diff
@@ -36,14 +38,27 @@ private function render($template, array $params)
 
     private function getConditionsToApprove()
     {
-        return [
-            'terms-and-conditions' => $this->l(
-                'I have read and agree to the terms and conditions.'
-            ),
-            'right-of-withdrawal' => $this->l(
-                'I have been informed about and agree with the conditions of exercise of my right of withdrawal for this order.'
+        $cms = new CMS(Configuration::get('PS_CONDITIONS_CMS_ID'), $this->context->language->id);
+        $link = $this->context->link->getCMSLink($cms, $cms->link_rewrite, (bool)Configuration::get('PS_SSL_ENABLED'));
+
+        $termsAndConditions = new TermsAndConditions;
+        $termsAndConditions
+            ->setText(
+                $this->l('I agree to the [terms of service] and will adhere to them unconditionally.'),
+                $link
             )
-        ];
+            ->setIdentifier('terms-and-conditions')
+        ;
+
+        $allConditions = Hook::exec('termsAndConditions', [], null, true);
```

Comment:
Hmm, how would you modify them? It's just a string and a few links, you can just return a new one with 'terms-and-conditions' as identifier and it will replace the default one.